src/lib/expansion/expansion.md

```diff
@@ -0,0 +1,79 @@
+Angular Material expansion provides a way to show and hide lightweight content, by collapsing and expanding a view with a nice animation. The `expanded` input allows to choose among the collapsed or expanded state. 
+
+Each expansion has a header and an action sections. The header section is always visible at the top of the component and contains a title and a description subsections. The action section is fixed at the bottom, being visible when the expansion is in expanded state
+
+When grouped by an `<md-accordion>` element, the expansions can be used for the creation of flows, as it brings up the possibility to expand one view at a time.
```

Comment:
The doc structure should be
1. Short, one-sentence summary of the component
2. Overview example
3. Expanded description of the component
4. Other component specific sections
5. Accessibility section